app/src/plotter.py

```diff
@@ -257,52 +271,91 @@ def fit_data(self):
+            self.plots[code].border_fill_color = self.config["background_fill_color"]
+
+            # Hide x grid
+            self.plots[code].xgrid.grid_line_color = None
+
+            # Configure axes
             self.plots[code].xaxis.ticker = x_range
-            self.plots[code].yaxis.formatter.use_scientific = False
+            self.plots[code].yaxis.ticker.desired_num_ticks = 8
+            self.plots[code].yaxis.minor_tick_line_color = None
+            self.plots[code].yaxis.major_tick_line_color = None
+            self.plots[code].xaxis.major_tick_line_color = None
+
+            for axis in [self.plots[code].xaxis, self.plots[code].yaxis]:
+                axis.axis_label_text_font = self.config["text"]["font"]
+                axis.axis_label_text_font_style = self.config["text"]["font_style"]
+                axis.major_label_text_font = self.config["text"]["font"]
+                axis.major_label_text_font_style = self.config["text"]["font_style"]
+                axis.axis_line_width = self.config["axis"]["axis_line_width"]
+                axis.axis_line_color = self.config["background_fill_color"]
 
             # Add lines to the plot
-            colors = Category20[20]
+            colors = overwrite_colors
             for i, line_name in enumerate(self.config["filters"]["multi_choice"]):
-                self.plots[code].line(x="x", y=line_name, source=self.fitted_data[code], color=colors[i], legend_label=line_name)
+                self.plots[code].line(
+                    x="x", y=line_name, source=self.fitted_data[code], color=colors[i], line_width=4)
+                self.plots[code].circle(
+                    x="x", y=line_name, source=self.fitted_data[code], color=colors[i], size=16)
+                self.plots[code].circle(
+                    x="x", y=line_name, source=self.fitted_data[code], color=self.config["background_fill_color"], size=12)
 
     def create_filters(self):
-        self.filters["single_choice"] = panel.widgets.RadioBoxGroup(
+        filters_single_choice = panel.widgets.RadioBoxGroup(
             name="Select unit", options=self.config["filters"]["single_choice"]
         )
 
-        # Create multi choice filters
-        self.filters["multi_choice"] = panel.widgets.CheckBoxGroup(
-            name="Select branches",
-            options=self.config["filters"]["multi_choice"],
-            value=self.config["filters"]["multi_choice"]
-        )
+        filters_multi_choice = panel.Column(*[
+            panel.Row(
+                panel.pane.HTML(
+                    '<div class="legend-field" style="background-color:{};"></div>'.format(color)),
+                panel.widgets.Checkbox(name=option, value=True,
+                                       css_classes=["legend-checkbox"])
+            )
+            for color, option in zip(overwrite_colors, self.config["filters"]["multi_choice"])
+        ])
+
+        self.filters["multi_choice"] = Accordion(
```

Comment:
As you use the same `header_color`, etc. for both filters, you can define these variables once before this line